.eslintrc

```diff
@@ -123,21 +124,15 @@
     {
       "files": ["*.tsx"],
       "rules": {
-        "react/prop-types": "off"
+        "react/prop-types": "off",
+        "react/jsx-props-no-spreading": "off"
```

Comment:
We now have full type coverage in `radiance-ui/src` 🥳 So we can turn this off for TS files.